Kickstarter-iOS.playground/Pages/VideoPlayer.xcplaygroundpage/Contents.swift

```diff
@@ -9,7 +9,8 @@ import UIKit
 let project = .cosmicSurgery
   |> Project.lens.video .~ (
     .template
-      |> Project.Video.lens.high .~ "https://d2pq0u4uni88oo.cloudfront.net/projects/1846844/video-562464-h264_high.mp4"
+      |> Project.Video.lens
+      .high .~ "https://d2pq0u4uni88oo.cloudfront.net/projects/1846844/video-562464-h264_high.mp4"
```

Comment:
This line break is a little strange, are we ok with swiftformat making changes like this? 🤔

Reply:
I didn't really check the playgrounds too closely - I can tweak this.